server/src/setup/server.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const { listeningPort } = require('./config');
+const animalRouter = require('../base/animal/animal.router');
+const getCorsConfig = require('./cors');
+
+module.exports = class Server {
```

Comment:
Can you provide some background on why you decided to structure your backend service this way?

Reply:
Main reason is that I like to keep my server setup organized so that whoever is reading my service, they can easily guide themselves around the project and also keeping middle-wares that will be attached onto the server instance can be its own modules so that it can be easily added onto and writing tests for smaller modules is much easier. @hughsoong